src/adapters/sharethrough.js

```diff
@@ -0,0 +1,149 @@
+var utils = require('../utils.js');
+var bidmanager = require('../bidmanager.js');
+var bidfactory = require('../bidfactory.js');
+
+const STR_BIDDER_CODE = "sharethrough";
+const STR_VERSION = "0.1.0";
+
+var SharethroughAdapter = function SharethroughAdapter() {
+
+  const str = {};
+  str.STR_BTLR_HOST = document.location.protocol + "//btlr.sharethrough.com";
+  str.STR_TEST_HOST =  document.location.protocol + "//btlr-prebid-test.sharethrough.com";
```

Comment:
Usually other bidders setup a test creative /placement on the live exchange. Is this possible?